src/Integration.Tests/python/test_source.py

```diff
@@ -0,0 +1,13 @@
+def get_mad_string() -> str:
```

Comment:
Include a doc string (triple quote) in here for good measure.

Reply:
Done with 8f9cd7d3b90865fd5991623790a0f3d61caed730.